src/osgEarth/LineDrawable.cpp

```diff
@@ -1159,9 +1159,17 @@ LineDrawable::setupShaders()
             vp->addBindAttribLocation("oe_LineDrawable_prev", LineDrawable::PreviousVertexAttrLocation);
             vp->addBindAttribLocation("oe_LineDrawable_next", LineDrawable::NextVertexAttrLocation);
             ss->getOrCreateUniform("oe_LineDrawable_limits", osg::Uniform::FLOAT_VEC2)->set(osg::Vec2f(-1,-1));
-            //_gpuStateSet->getOrCreateUniform("oe_GL_LineWidth", osg::Uniform::FLOAT)->set(1.0f);
+            //_gpuStateSet->getOrCreateUniform("oe_GL_LineWidth", osg::Uniform::FLOAT)->set(30.0f);
             //_gpuStateSet->getOrCreateUniform("oe_GL_LineStippleFactor", osg::Uniform::INT)->set(1);
             //_gpuStateSet->getOrCreateUniform("oe_GL_LineStipplePattern", osg::Uniform::INT)->set((int)~0);
+            
+#if defined(OSG_GLES3_AVAILABLE)
```

Comment:
The expectation here is that the coder will call GLUtils::setGlobalDefaults() someone else in the application, which initializes this uniform's default value.

Reply:
I think this may have come from me working on this at the same time as you were working on the line code. Now it’s settled down i will review.